packages/stargate/src/stargateclient.spec.ts

```diff
@@ -182,4 +198,79 @@ describe("StargateClient", () => {
       expect(balances).toEqual([]);
     });
   });
+
+  describe("broadcastTx", () => {
+    it("broadcasts a transaction", async () => {
+      pendingWithoutSimapp();
+      const client = await StargateClient.connect(simapp.tendermintUrl);
+      const wallet = await Secp256k1Wallet.fromMnemonic(faucet.mnemonic);
+      const [{ address, pubkey: pubkeyBytes }] = await wallet.getAccounts();
+      const publicKey = PublicKey.create({ secp256k1: pubkeyBytes });
+      const registry = new Registry();
+      const txBodyFields = {
```

Comment:
Great to have this working with all the details.

I bet these build steps could be replaced by a few helper functions and the whole process much easier to use.

Reply:
I imagine a lot of this will be abstracted away in `SigningStargateClient`.